bom/application/pom.xml

```diff
@@ -4764,11 +4764,7 @@
                 <groupId>org.jboss.logging</groupId>
                 <artifactId>jboss-logging-annotations</artifactId>
                 <version>${jboss-logging-annotations.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.jboss.logging</groupId>
-                <artifactId>jboss-logging-processor</artifactId>
-                <version>${jboss-logging-annotations.version}</version>
```

Comment:
This needs to stay around. It's used by the recent versions of the Maven compiler plugin to define the default version of the annotation processor.